admin/classes/rtbiz-hd-support/class-rtbiz-hd-support.php

```diff
@@ -452,13 +450,10 @@ public function submit_request() {
                                         <td>Email</td><td>' . strip_tags( $form_data['email'] ) . '</td>
                                     </tr>
                                     <tr>
-                                        <td>Website</td><td>' . strip_tags( $form_data['website'] ) . '</td>
-                                    </tr>
-                                    <tr>
-                                        <td>Subject</td><td>' . strip_tags( $form_data['subject'] ) . '</td>
+                                        <td>Details</td><td>' . strip_tags( $form_data['details'] ) . '</td>
                                     </tr>
                                     <tr>
-                                        <td>Details</td><td>' . strip_tags( $form_data['details'] ) . '</td>
```

Comment:
did we remove the details field?

Reply:
@milindmore22 

This change is from here https://github.com/rtCamp/rtbiz-helpdesk/pull/108/commits/eceb80df0147aa3c074ff8880488f46044e8469b

I'm not sure about the history of the change.